solutions/arithmetic/simple-interest.js

```diff
@@ -0,0 +1,20 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+let principal = parseInt(readLine());
+let time = parseInt(readLine());
+let rate = parseInt(readLine());
+
+let simpleInterest = (principal * time * rate)/100;
+
+console.log(parseInt(simpleInterest));
```

Comment:
This doesn't need conversion to int. I don't think there is a test case to check that scenario. Gives 100% score even without `parseInt`. Let us skip the conversion to keep it simple (as this is just first few days of bootcamp).

Reply:
Yes that's true. 

But keeping parseInt is making it consistent with other solutions..

If we remove parseInt, they need to know that js is auto-type-casting it from string to integer, which is relatively more difficult for them as of now..

Reply:
Removed.

Reply:
Removed.